css/properties/border-style.json

```diff
@@ -0,0 +1,56 @@
+{
+  "css": {
+    "properties": {
+      "border-style": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-style",
+          "support": {
+            "webview_android": {
+              "version_added": "2.6"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": "12"
```

Comment:
The original table notes `20 or earlier (12)` for the Edge browsers. I took that to mean `12` was the right number to include here, but I'm not certain about it. A closer review here would be appreciated.

Reply:
Yes, 12 is correct here.